content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -135,30 +134,26 @@ For pods, this is any pod that has a deletion timestamp set.
 
 ### Topology information {#topology}
 
-{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
+Each endpoint within an EndpointSlice can contain relevant topology information.
+The topology information includes the location of the endpoint and information
+about the corresponding Node and zone. This will be available in the following
+per endpoint fields on EndpointSlices:
+
+* `nodeName` - The name of the Node this endpoint is on.
+* `zone` - The zone this endpoint is in.
 
 {{< note >}}
-The topology field in EndpointSlices has been deprecated and will be removed in
-a future release. A new `nodeName` field will be used instead of setting
-`kubernetes.io/hostname` in topology. It was determined that other topology
-fields covering zone and region would be better represented as EndpointSlice
-labels that would apply to all endpoints within the EndpointSlice.
+In the v1 API, the per endpoint `topology` was effectively removed in favor of
+the dedicated fields `nodeName` and `zone`.
+
+Setting arbitrary topology fields per endpoint has been deprecated and will not
```

Comment:
```suggestion
Setting arbitrary topology fields per Endpoints has been deprecated and is not supported
```

Reply:
Agree that "endpoint" makes more sense in this case. (The naming of these two resources is rather confusing).

Reply:
I was suggesting using `Endppoints` because I am not sure if there is such resource called `endpoint`. The clarification makes sense. However, if the `endpoint` in this context means a field in the `EndpointSlice`, we may want to add backticks around 'endpoint` to make it clearer or more explicitly like the following:

> Setting arbitrary topology fields on the `endpoint` field of a `EndpointSlice` resource has been deprecated.